### PR TITLE
Add opctl integration tests

### PR DESCRIPTION
**Pull request by liudmylaru**
### Description

Moving opctl integration tests from BB to GH repo.
Jira: https://jira.oci.oraclecorp.com/browse/ODSC-39821

### What was done

- reviewed and fixed opctl tests
- added test_opctl_cli.py to integration tests
- added test_opctl_cmds.py to integration tests
- added test_opctl_conda.py to integration tests
- added test_opctl_utils.py to integration tests
- added test_opctl_core_site_cli.py to unittests
- updated cmds.py
- removed pytest.mark.opctl

### Pre-commit

```
check python ast.........................................................Passed
check docstring is first.............................(no files to check)Skipped
check json...........................................(no files to check)Skipped
check for merge conflicts................................................Passed
check yaml...........................................(no files to check)Skipped
detect private key.......................................................Passed
fix end of files.........................................................Passed
pretty format json...................................(no files to check)Skipped
trim trailing whitespace.................................................Passed
black....................................................................Passed
rst ``code`` is two backticks........................(no files to check)Skipped
rst ``inline code`` next to normal text..............(no files to check)Skipped
Detect hardcoded secrets.................................................Passed
check-copyright..........................................................Passed
```

### Validation

TeamCity testing build passed:
https://teamcity.oci.oraclecorp.com/buildConfiguration/Odsc_Ads_AdsCiCd_GhAdsIntegrationTestsPython38opctl/40777023?buildTab=log&focusLine=3910&logView=flowAware&linesState=485
image

### CODE REVIEW

**Review by mrDzurb — APPROVED** [review]

File: tests/integration/opctl/opctl_tests_files/notebook_test/exclude_check.py
Comment (qiuosier) [inline tests/integration/opctl/opctl_tests_files/notebook_test/exclude_check.py:1]: This files is probably generated by the test? I guess we don't need this file?

File: tests/integration/opctl/test_opctl_cmds.py
Comment (qiuosier) [inline tests/integration/opctl/test_opctl_cmds.py:1]: Seems like the tests here are similar to test_opctl_cli.py?

Comment (liudmylaru) [reply tests/integration/opctl/test_opctl_cmds.py:1]: yes, agree. This seems redundant to have both for integration tests. I can remove it now or we will clean it later. Please suggest the way to go. Thanks!

File: tests/integration/opctl/opctl_tests_files/notebook_test/exclude_check.py
Comment (liudmylaru) [reply tests/integration/opctl/opctl_tests_files/notebook_test/exclude_check.py:1]: will remove it

File: tests/integration/opctl/test_opctl_cmds.py
Comment (liudmylaru) [reply tests/integration/opctl/test_opctl_cmds.py:1]: Discussed and agreed that cLI tests are enought to cove functionality in integration tests. Will remove this file

**Review by mrDzurb — APPROVED** [review]

**Review by qiuosier — APPROVED** [review]

**Pull request merged**